src/components/DisplayQuote.js

```diff
@@ -0,0 +1,50 @@
+import React, { useState, useEffect } from 'react';
+import '../styles/display-quote.css';
+
+const JokeComponent = () => {
+  const [joke, setJoke] = useState(null);
+  const [loading, setLoading] = useState(true);
+  const [error, setError] = useState(false);
+
+  useEffect(() => {
+    const myAPIKey = 'Ixgm2cYnlo7wGSvmiMQCVvfUJ6kVeTvgNZShaXEC';
+    const apiURL = 'https://api.api-ninjas.com/v1/dadjokes?limit=1';
+
+    const options = {
+      method: 'GET',
+      headers: { 'X-Api-Key': myAPIKey },
+    };
+
+    const fetchData = async () => {
+      try {
+        const response = await fetch(apiURL, options);
+        const data = await response.json();
+        setJoke(data[0].joke);
+      } catch (error) {
+        setError(true);
+      }
+      setLoading(false);
+    };
+
+    fetchData(); // Call the function to fetch data when the component mounts
```

Comment:
- [ ] I suggest that you change a function to `Quote` not `Joke` as the requirement says
- [ ] Please change the API from `Joke` to `Quote` you can refer to this [link](https://api-ninjas.com/api/quotes)